pkg/apis/batch/v2alpha1/defaults_test.go

```diff
@@ -20,179 +20,58 @@ import (
 	"reflect"
 	"testing"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/kubernetes/pkg/api"
 	_ "k8s.io/kubernetes/pkg/api/install"
-	"k8s.io/kubernetes/pkg/api/v1"
 	_ "k8s.io/kubernetes/pkg/apis/batch/install"
 	. "k8s.io/kubernetes/pkg/apis/batch/v2alpha1"
 )
 
-func TestSetDefaultJob(t *testing.T) {
-	defaultLabels := map[string]string{"default": "default"}
+func TestSetDefaultCronJob(t *testing.T) {
```

Comment:
Yeah, I've realized we didn't have one for CronJobs, I should've put that in a separate commit, will do it next time.